### DISCUSSION TITLE
"clone class python environment" not working

### DISCUSSION THREAD

**Discussion by allenmoench** [earthlab-education/Earth-Analytics-AY24#570] [Q&A] [answered]
Hello! I'm working on getting the python environment set up on my PC. I'm in github desktop trying to clone the repository from the URL provided in the instructions ("week 7 agenda" document), and I get this message. Is that repository unavailable for some reason?
![image](https://github.com/user-attachments/assets/a27c8d31-e71a-4d6f-a25f-cc0153ba24a2)

**Comment by allenmoench** [comment]
Nevermind, I got that one figured out! Seems to have been an issue with copy/pasting the hyperlinked URL - when I typed it instead, it worked.

Now I'm stuck at a different spot. When I type "cd" and then paste the path to the repository, it says "no such file or directory exists" 
![image](https://github.com/user-attachments/assets/96d4fbd8-adb5-41be-9694-874cc52adcf2)

**Reply by akpost21** [reply, marked as answer]
@allenmoench Hello! I believe your slashes are the wrong direction. Use a forward slash (/) instead of a backslash (\) in your directory path. See if that works. Windows uses the opposite direction slash than most programs use annoyingly....

**Reply by allenmoench** [reply]
That worked, thanks!!

**Reply by akpost21** [reply]
Yay, that's great!

**Reply by eculler** [reply]
@allenmoench I recommend using the `TAB` key to autocomplete paths in the command line -- helps to catch typos earlier!
